### PR TITLE
Add assets for 'TRO-94c925'

### PR DESCRIPTION
**Pull request by NeltudOne**

### CODE REVIEW

File: tokens/TRO-94c925/info.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "website": "https://nelsontuduriprods.wixsite.com/tableauxnelsontuduri",
```
Comment (NeltudOne) [inline tokens/TRO-94c925/info.json:2]: suppression of old link

**Review by NeltudOne — COMMENTED** [review]
new website replace old link

**Review by NeltudOne — COMMENTED** [review]
change logo

**Review by NeltudOne — COMMENTED** [review]
replace website info

**Review by paulmultiversx — APPROVED** [review]

**Pull request merged**